test/test_matmul_cuda.py

```diff
@@ -1758,17 +1780,15 @@ def scaled_grouped_mm_helper(self, alist, blist, ascalelist, bscalelist, outlist
     # combinations.
 
     @unittest.skipIf(TEST_WITH_ROCM, "ROCm doesn't support CUTLASS")
-    @xfailIfSM100OrLater
     @unittest.skipIf(not SM90OrLater, "Grouped gemm supported on SM90")
     @parametrize("fast_accum", [False, True])
     @parametrize("strided", [False, True])
     def test_scaled_grouped_gemm_2d_2d(self, fast_accum, strided):
         device = "cuda"
-        m, n, k, n_groups = 16, 32, 64, 4
+        m, n, k, n_groups = 128, 256, 384, 4
```

Comment:
Is this due to new shape restrictions? If so might be worth leaving old shapes as well with an assert fail regex context manager: https://github.com/pytorch/pytorch/blob/2df2e3bb511eb3d72742334b116e97656539570d/torch/testing/_internal/common_utils.py#L4218